com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GlyphBankBuilder.java

```diff
@@ -41,8 +42,13 @@ public Task create(IResource input) throws IOException, CompileExceptionError {
         ProtoUtil.merge(input, fontDescbuilder);
         FontDesc fontDesc = fontDescbuilder.build();
 
+        File file = new File(fontDesc.getFont());
+        String fileNameWithExtension = file.getName();
+        int dotIndex = fileNameWithExtension.lastIndexOf(".");
+        String fileNameWithoutExtension = (dotIndex == -1) ? fileNameWithExtension : fileNameWithExtension.substring(0, dotIndex);
```

Comment:
we can't use `.font` filename, because then we will have different tiles all the time. But instead we use the source filename and instead or 

>/_generated_76b310fccd3c24f7.glyph_bankc

it is

>/_generated_bmfont_76b310fccd3c24f7.glyph_bankc

Reply:
Sure, it's possible that we generate a few banks for the same source font. But we can't show this connection by file names. The only way to know what's connects to what is `graph.json` in the build folder

Reply:
So what happens when you have two fonts with the same settings (i.e same hash), but different filenames? We just include one of them in the build?

Reply:
how it's possible? why it's two separate fonts if their hashes are identical?

Reply:
this is the name of the initial font not a font file
so, if a font file refers to `my_font.ttf`, then glyph bank will be 
`_generated_my_font_hash.glyph_bankc`